lib/msal-angular/src/msal.service.ts

```diff
@@ -59,6 +62,14 @@ export class MsalService implements IMsalService {
         .then(() =>
           this.instance.handleRedirectPromise(hash || this.redirectHash)
         )
+        .catch((error: AuthError) => {
+          this.logger.error("Error while executing handleRedirectPromise");
```

Comment:
I am not sure if you need the `catch` block here with no additional error payload in the logs.

Reply:
Yeah, it is kind of redundant. I can remove it.